### ISSUE TITLE
Felix code coverage should exclude "*test*" files

### ISSUE THREAD

**Issue by plwhite** [Metaswitch/calico#99]
These are removed by nosetests but not by the coverage tool itself. This is just a change to `.coveragerc` (the checked in version needs an extra `omit` value).

**Comment by Lukasa** [comment]
Resolved by #100.

**Comment by Lukasa** [comment]
Sadly, it seems that the change in #100 was overly aggressive and turned off *all* test coverage.

**Comment by Lukasa** [comment]
The correct coveragerc looks like this:

```
[run]
include = *felix*
omit = *stub_*
       calico/felix/test/*
branch = True
```

**Comment by plwhite** [comment]
Well, that is rather embarrassing. I tested it, honest, but I obviously tested it badly since retesting it does not seem to work. I'll fix it.

**Comment by plwhite** [comment]
Fixed by pull request #102
